components/blitz/src/ome/services/blitz/measurements/Helper.java

```diff
@@ -31,8 +29,8 @@
      * Factory method to generate Helper instances from the LSID of a given
      * column.
      * 
-     * @param lsid
-     * @return
+     * @param lsid The lsid
+     * @return See above.
```

Comment:
similar to tag work, guideline evolved as we were going 
and also due to the style of people "cleaning" the doc
Another round will be required for header